pkg/apis/crd/v1alpha2/types.go

```diff
@@ -543,3 +543,33 @@ type TrafficControlList struct {
 
 	Items []TrafficControl `json:"items"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeLatencyMonitor is used to monitor the latency between nodes in a Kubernetes cluster. It is a singleton resource,
+// meaning only one instance of it can exist in the cluster.
+type NodeLatencyMonitor struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec NodeLatencyMonitorSpec `json:"spec"`
+}
+
+type NodeLatencyMonitorSpec struct {
+	// PingInterval specifies the interval between ping requests.
+	// Ping interval should be greater than or equal to 1s(one second).
+	// Defaults to "10". Valid time units are "s".
+	PingInterval int `json:"pingInterval"`
```

Comment:
Also, the field should be optional and default to something reasonable (60s ?)

Reply:
That makes sense, I will change it.